test/parallel_api/ranges/std_ranges_test.h

```diff
@@ -22,6 +22,8 @@
 
 #if _ENABLE_STD_RANGES_TESTING
 
+static_assert(ONEDPL_HAS_RANGE_ALGORITHMS >= 202409L);
```

Comment:
Why this `static_assert` has placed under `#if _ENABLE_STD_RANGES_TESTING` check?

Reply:
Because the feature is available and the feature macro is defined only under similar conditions.

Reply:
ok